test/runtests.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# runtests.sh
+
+# create bin directory if it doesn't exist
+if [ ! -d "../bin" ]
+then 
+	mkdir ../bin
+fi
+
+
+# delete output from previous run
+rm actual.txt
```

Comment:
You might want to check if the file exists here. Also 1 blank line above should be enough I suppose.